File: src/main/java/min_max_sum/MinMaxSum.java
package min_max_sum;

import java.util.Arrays;

public class MinMaxSum {
    public void minMaxSum(){
        int max = 0;
        int min = 0;
        int[] list = new int[]{1,2,3,4,5};
        Arrays.sort(list);

        for(int i = 0; i<list.length-1;i++){
            min = min + list[i];
        }
        System.out.println(min);


        for(int i = list.length-1; i>0; i--){
            max = max + list[i];
        }

        System.out.println(max);
    }
}

File: src/main/kotlin/min_max_sum/minMaxSum.kt
package min_max_sum

import java.util.*


fun minMaxSum() {
    var max = 0
    var min = 0
    val list = intArrayOf(1, 2, 3, 4, 5)
    Arrays.sort(list)

    for (i in 0 until list.size - 1) {
        min += list[i]
    }
    println(min)

    for (i in list.size - 1 downTo 1) {
        max += list[i]
    }
    println(max)
}